stdlib/uchar.mli

```diff
@@ -143,6 +143,20 @@ val utf_decode_invalid : int -> utf_decode
     smaller or equal to [4] (this is not checked by the module). The
     resulting decode has {!rep} as the decoded Unicode character. *)
 
+val utf_8_decode_length_of_byte : char -> int
+(** [utf_8_decode_length_of_byte byte] is the number of bytes, from 1
+    to {!max_utf8_decode_length}, that a valid UTF-8 decode starting
+    with byte [byte] would consume or [0] if [byte] cannot start a
+    valid decode.
```

Comment:
Thanks this rewording address my fear that a reader might be surprised that 
`utf_8_decode_length_of_byte s.(0) (* ≠ 0 *)` might be different from `Uchar.decode_length (String.get_utf_8_uchar s 0)`.
Explaining the `0` exceptional case at the end is also nice.

Reply:
Yes that became my worry too.